test/ctia/test_helpers/es.clj

```diff
@@ -207,7 +207,7 @@
            :_id _id)))
 
 (defn load-bulk
-  ([conn docs] (load-bulk conn docs "true"))
+  ([conn docs] (load-bulk conn docs "wait_for"))
```

Comment:
this ensure that the request actually `wait_for` the index refresh and does not only triggers it.